src/bdmsupport.c

```diff
@@ -471,22 +471,12 @@ static int bdmCheckVMC(char *name, int createSize)
     return sysCheckVMC(bdmPrefix, "/", name, createSize, NULL);
 }
 
-static void bdmGetAppsPath(char *path, int max)
+static void bdmGetPrefix(char *path, int max)
 {
-    snprintf(path, max, "%sAPPS", bdmPrefix);
-}
-
-static void bdmGetLegacyAppsPath(char *path, int max)
-{
-    snprintf(path, max, "%sconf_apps.cfg", bdmPrefix);
-}
-
-static void bdmGetLegacyAppsInfo(char *path, int max, char *name)
-{
-    snprintf(path, max, "%sCFG/%s.cfg", bdmPrefix, name);
+    snprintf(path, max, bdmPrefix);
 }
```

Comment:
I like the reduction, but this is a strange way to "Get" a string. More simple would be something like:
`static char * bdmGetPrefix() {return bdmPrefix;}`

Or instead of a function in `item_list_t`, this coult also be a variable (like `char *prefix;`) in `item_list_t`.